package.json

```diff
@@ -66,7 +65,7 @@
     "express": "4.18.2",
     "express-async-errors": "3.1.1",
     "express-session": "^1.17.3",
-    "govuk-frontend": "4.3.1",
+    "govuk-frontend": "4.5.0",
```

Comment:
How come we've changed this? Are we going to run full regression in this case?

Reply:
This was so that the spinner could use all the govuk colour variables, with 4.3.1 the code wouldn't build and threw this error:
Error: Undefined variable.
   ╷
13 │ $govuk-button-colour: $govuk-button-background-colour; // Line 20
   │                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reply:
Alright cool! Just raising because package versions have caused issues with the tests yesterday so just something to bear in mind when testing @Z-Mohamoud